charts/console/Chart.yaml

```diff
@@ -3,14 +3,19 @@ name: console
 description: >-
   deploys the plural console and additional dependencies, for use in bring-your-own-kube setups
 appVersion: 0.8.15
-version: 0.1.40
+version: 0.2.0
 dependencies:
   - name: kas
     version: 0.0.5
     repository: file://../../plural/helm/kas
   - name: controller
     version: 0.0.21
     repository: file://../controller
+  - name: kubernetes-dashboard
+    alias: dashboard
+    version: 7.0.1
+    repository: https://kubernetes.github.io/dashboard
+    condition: kubernetes-dashboard.enabled
```

Comment:
shouldn't this be `dashboard.enabled`?

Reply:
Yep, it should. Added this before alias and forgot to update.